templates/settings-admin.php

```diff
@@ -227,27 +227,46 @@ function FormatMegabytes(int $byte): string {
 
 			<div class="col col-12">
 				<div class="row">
-					<div class="col col-4 col-l-6 col-m-12">
+					<div class="col">
 						<div class="infobox">
-							<div class="interface-wrapper">
-								<?php if ($_['storage']['num_users'] > 0) : ?>
-									<?php p($l->t('Total users:')); ?>
-									<span class="info"><?php p($_['storage']['num_users']); ?></span><br>
+							<div class="interface-wrapper active-users-wrapper">
+								<?php if ($_['activeUsers']['last1hour'] > 0) : ?>
+									<div class="active-users-box">
+										<?php p($l->t('1 Hour Active Users')); ?><br>
+										<span class="info"><?php p($_['activeUsers']['last1hour']) ?></span><br>
+										<em><?php p($l->t('%s%% of all users', [round($_['activeUsers']['last1hour'] * 100 / $_['storage']['num_users'], 1)])) ?></em>
+									</div>
 								<?php endif; ?>
 
 								<?php if ($_['activeUsers']['last24hours'] > 0) : ?>
-									<?php p($l->t('24 hours:')); ?>
-									<span class="info"><?php p($_['activeUsers']['last24hours']) ?></span><br>
+									<div class="active-users-box">
+										<?php p($l->t('1 Day Active Users')); ?><br>
+										<span class="info"><?php p($_['activeUsers']['last24hours']) ?></span><br>
+										<em><?php p($l->t('%s%% of all users', [round($_['activeUsers']['last24hours'] * 100 / $_['storage']['num_users'], 1)])) ?></em>
+									</div>
 								<?php endif; ?>
 
-								<?php if ($_['activeUsers']['last1hour'] > 0) : ?>
-									<?php p($l->t('1 hour:')); ?>
-									<span class="info"><?php p($_['activeUsers']['last1hour']) ?></span><br>
+								<?php if ($_['activeUsers']['last7days'] > 0) : ?>
+									<div class="active-users-box">
+										<?php p($l->t('7 Day Active Users')); ?><br>
+										<span class="info"><?php p($_['activeUsers']['last7days']) ?></span><br>
+										<em><?php p($l->t('%s%% of all users', [round($_['activeUsers']['last7days'] * 100 / $_['storage']['num_users'], 1)])) ?></em>
+									</div>
+								<?php endif; ?>
+
+								<?php if ($_['activeUsers']['last1month'] > 0) : ?>
+									<div class="active-users-box">
+										<?php p($l->t('1 Month Active Users')); ?><br>
+										<span class="info"><?php p($_['activeUsers']['last1month']) ?></span><br>
+										<em><?php p($l->t('%s%% of all users', [round($_['activeUsers']['last1month'] * 100 / $_['storage']['num_users'], 1)])) ?></em>
+									</div>
 								<?php endif; ?>
 
-								<?php if ($_['activeUsers']['last5minutes'] > 0) : ?>
-									<?php p($l->t('5 mins:')); ?>
-									<span class="info"><?php p($_['activeUsers']['last5minutes']) ?></span><br>
+								<?php if ($_['storage']['num_users'] > 0) : ?>
+									<div class="active-users-box">
+										<?php p($l->t('Total users')); ?><br>
```

Comment:
I feel like somehow the *Total users* should be separated out somehow since it's a bit of a different stat versus all the *Active Users* ones. It's more "Users that have ever logged in and still have an account in the database*. And it's a timeless stat so "Total" doesn't seem like the correct wording here.

Maybe just relabeling it *Users That Have Ever Logged In* but there must be a shorter phrase than I'm coming up with at the moment. :thinking:

Reply:
True. 

Total Users here are all users who logged in at least once. 

Okay for me to remove total users.

Reply:
3 propositions : 
- simply rename to « All users »
- only show the number of users in the percentage subline (something like « xx% of all yyyy users », or « xx% of all users (yyyy) ». But this make this statistic not very visible.
- Move this stat to another block, with the possibility to differentiate between users who never logged once

Reply:
For now, I suggest we take this out and add an issue for thorough planning. I think we need a background job to fetch and cache the correct user count because looping over all backends is expensive. That only works for database/ldap backend, for saml/oidc we only know how many users were logged in once.

Reply:
> I suggest we take this out

You mean remove the metric from this page altogether or simply show it elsewhere?

Reply:
Done